splunklib/binding.py

```diff
@@ -453,6 +455,11 @@ class Context(object):
     :type splunkToken: ``string``
     :param headers: List of extra HTTP headers to send (optional).
     :type headers: ``list`` of 2-tuples.
+    :param retires: Number of retries for each HTTP connection (optional, the default is 0).
+                    NOTE THAT THIS MAY INCREASE THE NUMBER OF ROUND TRIP CONNECTIONS TO THE SPLUNK SERVER.
```

Comment:
One minor note, it's probably worth noting that this is also done in a blocking/synchronous way, so maybe amend this to:
```
NOTE THAT THIS MAY INCREASE THE NUMBER OF ROUND TRIP CONNECTIONS TO THE SPLUNK SERVER,
AND BLOCK THE CURRENT THREAD WHILE RETRYING.
```